components/brave_extension/extension/brave_extension/background/reducers/shieldsPanelReducer.ts

```diff
@@ -127,6 +128,16 @@ export default function shieldsPanelReducer (
         .updateTabShieldsData(state, tabId, { braveShields: action.setting })
       break
     }
+    case shieldsPanelTypes.REPORT_BROKEN_SITE: {
+      const tabId: number = shieldsPanelState.getActiveTabId(state)
+      const tabData = shieldsPanelState.getActiveTabData(state)
+      if (!tabData) {
+        console.error('Active tab not found')
+        break
+      }
+      reportBrokenSite(tabId)
+      break
```

Comment:
I think I was just using it as a validity check. Is there a better way to do that?

Or would you say `chrome.tabs` is safe to use without a check?

Reply:
chrome.tabs will return something you can trust - either there will be an active tab in the current window, or there won't

Reply:
Unless I'm doing something wrong, I don't think the `chrome.tabs` API can work here.  `chrome.tabs.getCurrent()` returns `undefined`, I suspect because it considers the "current" context to be the Shields panel itself.

Reply:
@antonok-edm you would use:
```
chrome.tabs.query({active: true, currentWindow: true }, function (tabs) {
  ...
})
```

For example, this works for me from dev tools of shields panel:
```
chrome.tabs.query({active: true, currentWindow: true }, console.log)
```

Reply:
That worked for me, just updated it